bundle/src/test/java/com/adobe/acs/commons/filefetch/impl/FileFetchImplTest.java

```diff
@@ -76,6 +77,8 @@ protected HttpURLConnection openConnection() throws IOException {
   }
 
   @Test
+  @Ignore
+  // Ignore until https://wcm-io.atlassian.net/browse/WTES-48 is resolved
```

Comment:
@joerghoh 2.7.2 is released now.

https://mvnrepository.com/artifact/io.wcm/io.wcm.testing.aem-mock.junit5

Reply:
thanks, will continue later this week.